chain-signatures/node/src/mesh/connection.rs

```diff
@@ -108,6 +115,10 @@ impl Pool {
 
         let mut potential_active = self.potential_active.write().await;
         *potential_active = Some((participants.clone(), Instant::now()));
+        tracing::debug!(
+            "potential participant account id's {:?}",
+            participants.account_ids()
+        );
```

Comment:
I'd say we should merge logging the potential participants and active participants into one call in mesh, because this ping call happens every 300ms. It's going to be really hard to find what we want

Reply:
we can get rid of these now since https://github.com/near/mpc/pull/800 already does what I mentioned